src/CMakeLists.txt

```diff
@@ -272,7 +272,8 @@ ADD_DEFINITIONS(-DEIGEN_MPL2_ONLY)
 # - external if CERES_DIR_HINTS and FIND_PACKAGE return a valid Ceres setup
 # ==============================================================================
 MESSAGE(STATUS "Trying to find package Ceres for openMVG: ${CERES_DIR_HINTS}")
-FIND_PACKAGE(Ceres QUIET HINTS ${CERES_DIR_HINTS})
+FIND_PACKAGE(Ceres QUIET HINTS ${CERES_DIR_HINTS}
+                   REQUIRED SuiteSparse)
```

Comment:
ah ok sorry, my ceres is not up-to-date. Anyway I think the correct way to do that is to use `COMPONENTS` as explained here https://github.com/ceres-solver/ceres-solver/blame/master/cmake/CeresConfig.cmake.in#L318

Another thing is that we may want to allow people to use ceres without SS? Maybe we can put this `find_package` in a if:

```cmake
set(OPENMVG_REQUIRE_CERES_WITH_SUITESPARSE ON)
...
if(OPENMVG_REQUIRE_CERES_WITH_SUITESPARSE)
  FIND_PACKAGE(Ceres REQUIRED HINTS ${CERES_DIR_HINTS} COMPONENTS
else()
 FIND_PACKAGE(Ceres QUIET HINTS ${CERES_DIR_HINTS}
endif()
```